index.js

```diff
@@ -9,6 +9,12 @@ var IncludeAll = require('./lib/include-all.js');
 
 module.exports = {
 
+  _getStyleFunnel: function() {
+    return new Merge([this._getPodStyleFunnel(), this._getClassicStyleFunnel()].filter(Boolean), {
```

Comment:
both of these methods will always return something it looks like. Is the `filter(Boolean)` necessary?